basic-spring-boot-tekton/.openshift/templates/deployment.yml

```diff
@@ -16,7 +16,11 @@ objects:
     annotations:
       description: The web server's http port.
     labels:
-      app: ${APPLICATION_NAME}
+      app.kubernetes.io/name: ${APPLICATION_NAME}
+      app.kubernetes.io/instance: ${APPLICATION_NAME}-build
```

Comment:
Should these be `${APPLICATION_NAME}-deployment` in this file?

Reply:
yes it should.

Reply:
updated